README.md

```diff
@@ -1,5 +1,208 @@
+
+- **Service Registry** (Consul from Hashicorp): An active service registry for mapping API specification identifiers and endpoints, used to establishing right connections for wanted information exchange.
+
+- **Service Explorer**: List and view approved API specifications, published within the API specification registry, together with active provider API instances, published within the service registry. The `Sinetiq Service Explorer` is a VSCode extension that helps you find service instances, their specifications and verify the implementaition while developing.
+
+<p>&nbsp;</p>
+<div style="text-align:center">
+<img align="center" src="./dosc/media/../../docs/media/api-governance-processes.png" width="600">
+</div>
+  
+<p>&nbsp;</p>
+
+### Installation Guide
+
+#### Registry services
+This will start the registry components, `API specification registry` and `Service Registry`.
+
+1. Fetch the demonstration github repository.   
+    ```sh
+    git clone git@github.com:Sinetiq/api-specification-governance.git
+    ```
+
+2. Change directory to the root git folder where the docker-compose.yml can be found, run the docker command.
+    ```sh
+    cd api-specification-governance
+
+    docker-compose up
+    ```
+
+_You can stop your registry services with CTRL-C._
+
+#### Service explorer
+Download and configure Visual Studio Code as your demonstration editor. Install the SINETIQ Service Explorer plugin. Follow the readme at the visual studio code marketplace [https://marketplace.visualstudio.com/items?itemName=Sinetiq.service-explore](https://marketplace.visualstudio.com/items?itemName=Sinetiq.service-explore).
+
+#### CLI tool: Manual run (not devops pipeline)
+
+1. Download CLI tool component from [https://www.npmjs.com/package/@sinetiq/api-specification-registry-cli](https://www.npmjs.com/package/@sinetiq/api-specification-registry-cli). Configure the CLI for your GIT repository containing API specifications (YAML files).
+   
+```sh
+npm i @sinetiq/api-specification-registry-cli
+```
+
+#### CLI tool: Automation, devops (CI/CD) pipeline
+
+1. Use `pipeline example` matching your environment.
+
+2. Create the API specification and version control it in your git-repo, attached to your created pipeline.
+
+3. Create a Pull Request/Merge Request, review, approve and merge the API specification, initiate the pipeline.
+
+
+_Your demonstration environment is now ready._
+
+#### Verify installation
+
+1. Verify the API Specification Registry.  
+   Open a webbrowser, use url http://localhost:3333/ to verify up-n-running.
+
+2. Verify Service Registry.  
+   Open a webbroser, use url http://localhost:8500/ to verify up-n-running.
+
+3. Verify Service Explorer.  
```

Comment:
We should add (or link but then emphasise) how to configure the plugin to go towards your local environment instead of the global one. Once we have the configuration file for the plugin done, then this will be automatic when checking out the demo repository though.

Reply:
How?